src/views/CreateNew/DeployView.vue

```diff
@@ -223,6 +231,18 @@
     tracker.create.eventDeployed(solutionStore.solution).track()
   }
 
+  const minutes = computed(() => {
+    return Math.floor(timer.value / 60)
+  })
+  
+  const seconds = computed(() => {
+    return timer.value % 60
+  })
+
+  const isGreaterThanMinute = computed(() => {
+    return timer.value > 60
+  })
+
```

Comment:
pergunta: porque não ter toda a lógica em uma só computed e já retornar o valor formatado para o template? A solução não está complexa de entender, porém estamos abrindo 3 pontos de reatividade.

```suggestion
  const MINUTE_IN_SEC = 60 
  const elapsedTime = computed(() => {
    const isGraterThanMinute = timer.value > MINUTE_IN_SEC
  
    if (!isGraterThanMinute) {
      return `Project started {{ timer.value }}s ago`
    }
    
    const seconds = timer.value % MINUTE_IN_SEC
    const minutes = Math.floor(timer.value / MINUTE_IN_SEC)
    
    return `Project started {{ minutes }}m{{ seconds }}s ago`
  })


```

Reply:
Interessante! Sou mais favorável a sugestão do Will de usar apenas uma computed.
Parece um pouco mais contido o escopo